### PR TITLE
Bugfix: Health Check Dashboard

### PR DESCRIPTION
**Pull request by loivsen**
Fixes some loading issue on the health check dashboard

### CODE REVIEW

File: src/packages/health-check/dashboard-health-check.element.ts [hunk 1]
```diff
@@ -58,7 +59,7 @@ export class UmbDashboardHealthCheckElement extends UmbLitElement {
 				weight: 500,
 				meta: {
 					label: group.name || '',
-					api: () => import('./health-check.context.js'),
+					api: UmbHealthCheckContext,
```
Comment (iOvergaard) [inline src/packages/health-check/dashboard-health-check.element.ts:62]: Now that you are using the `loadManifestApi()` function, I think it should work if you leave this as a dynamic import, right?

File: src/packages/health-check/health-check.context.ts [hunk 2]
```diff
@@ -43,6 +44,10 @@ export class UmbHealthCheckContext {
 			this._results.next(undefined);
 		}
 	}
+
+	public destroy(): void {
+		this.destroy();
+	}
```
Comment (iOvergaard) [inline src/packages/health-check/health-check.context.ts:50]: Just leave this function empty - currently it invokes itself :)

```suggestion
	public destroy(): void {
	}
```

File: src/packages/health-check/dashboard-health-check.element.ts (hunk 1 above)
Comment (iOvergaard) [inline src/packages/health-check/dashboard-health-check.element.ts:62]: In order to stay aligned with other manifests, could we then move the `api` property out of the "meta" object and onto the manifest itself?

**Review by iOvergaard — CHANGES_REQUESTED** [review]
Looks good, left a few comments :)

File: src/packages/health-check/dashboard-health-check.element.ts (hunk 1 above)
Comment (loivsen) [reply src/packages/health-check/dashboard-health-check.element.ts]: Yeah last thing that I forgot to add 🚀

**Review by iOvergaard — APPROVED** [review]

**Pull request merged**